### PR TITLE
New command added dump-hosts.

### PR DESCRIPTION
**Pull request by r3wald**
I was looking for some way to update the list of changes in a non-interactive way. Therefore I crated a new command: dump-hosts. This creates a JSON file which is suitable for feeding to sync-file.

### CODE REVIEW

**Comment by freekmurze** [comment]
While the code looks good, the tests are failing. Code you take a look at that?

**Comment by r3wald** [comment]
Done.

File: tests/Models/Concerns/HasProcessTest.php [hunk 1]
```diff
@@ -25,7 +25,7 @@ public function it_uses_the_host_in_the_process_command()
     {
         $this->check->getProcessCommand();
 
-        $this->assertStringStartsWith("ssh my-host   'bash", $this->check->getProcessCommand());
+        $this->assertStringStartsWith("ssh  my-host   'bash", $this->check->getProcessCommand());
```
[the remark this answers is not in the document]
Comment (r3wald) [reply tests/Models/Concerns/HasProcessTest.php]: Well, that would have meant much more changes to the code. Who on earth committed these tests in the first place? It's obvious they never succeeded.
Anyway, I will look into it.

**Comment by r3wald** [comment]
#93 only fixes the tests, already merged into this branch.

**Comment by freekmurze** [comment]
Thanks!

**Comment by freekmurze** [comment]
Fyi: I cleaned up the PR a little so the code is more clear: https://github.com/spatie/laravel-server-monitor/commit/bbfa11cdab9575f58a183dc8d193ca51a5f04e37

**Comment by r3wald** [comment]
That's ok with me. If remember correctly some test will fail when "ip" and "port" are always dumped.

**Pull request merged**